org.electrum.electrum.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.electrum.electrum</id>
+  <name>Electrum</name>
+  <developer_name>spesmilo</developer_name>
+  <launchable type="desktop-id">org.spesmilo.Electrum.desktop</launchable>
```

Comment:
this id doesn't match. should it be `org.electrum.electrum.desktop`?

Reply:
Yes, you are correct.